client/sources/common/interpreter.py

```diff
@@ -31,6 +33,15 @@ def __init__(self, console, setup='', teardown='', **fields):
         self.setup = setup
         self.teardown = teardown
 
+        # must reload for fpp problems
```

Comment:
For normal okpy problems, `python3 ok -q q1`, ok needs to import the student code for q1 exactly once, before terminating the process. For fpp problems, ok needs to import the student code and test it multiple times without terminating the process (and a simple import <filename> [no longer works](https://stackoverflow.com/questions/437589/how-do-i-unload-reload-a-python-module)), since the student needs to be able to adjust their code and re-test it.

Reply:
ah i see, okay